pkg/authorization/authorizer/scope/converter.go

```diff
@@ -119,9 +119,12 @@ const (
 	UserInfo        = UserIndicator + "info"
 	UserAccessCheck = UserIndicator + "check-access"
 
+	// UserListProject gives explicit permission to see the projects that this token can see.
+	UserListProject = UserIndicator + "list-projects"
+
 	// UserListProject gives explicit permission to see the projects a user can see.  This is often used to prime secondary ACL systems
 	// unrelated to openshift and to display projects for selection in a secondary UI.
-	UserListProject = UserIndicator + "list-projects"
+	UserListAllProjects = UserIndicator + "list-all-projects"
```

Comment:
> I think most people would expect user:list-projects to let you list all the user's projects.

I'll also point out that most people will probably be using for cases where they think their integration is "trusted", so getting back a project list where you don't have access to the projects sucks.  Imagine the web-console as a for instance.  

Still updating the names though.